src/main/java/com/rewedigital/composer/configuration/DefaultConfiguration.java

```diff
@@ -10,14 +10,14 @@
 
 public class DefaultConfiguration {
 
-    private static final Map<String, Object> defaultConfiguration = initDefaultConfigurationMap();
+    private static final Map<String, Object> DEFAULT_CONFIGURATION = initDefaultConfigurationMap();
```

Comment:
During the last years, I have grown the feeling that the ALL_CAPS convention for `private static final` fields sometimes decreases readability. For example, In this particular case I don't think that the constant aspect of the field is important. It's just an optimization and should not be overemphasized. Thus I liked the normal camelCase naming more

Reply:
Let's talk in person on Monday about that. Reverted the change.